### ISSUE TITLE
Clean up the naming of examples

### ISSUE THREAD

**Issue by podhrmic** [rems-project/cn-tutorial#26]
The tutorial example names are confusing - it goes as `0.c`, `slf1_basic_example.signed.c` and so on.

Standardize the names, e.g. `example_1.c`...`example_N.c`, possibly with added explanation such as `example_4_abs.c` etc.

**Comment by bcpierce00** [comment]
Agreed that it is a mess at the moment.  I'd rather have meaningful alphabetic names, though (i.e., abs.c, not example_4_abs.c), rather than number things in a single big sequence (which is much harder to manage as things get added, moved around, etc.).
